validator-api/src/node_status_api/routes.rs

```diff
@@ -211,14 +211,21 @@ pub(crate) async fn get_mixnode_inclusion_probability(
     identity: String,
 ) -> Json<Option<InclusionProbabilityResponse>> {
     let mixnodes = cache.mixnodes().await;
+    let rewarding_params = cache.epoch_reward_params().await.into_inner();
+
+    if mixnodes.len() <= rewarding_params.rewarded_set_size() as usize {
+        return Json(Some(InclusionProbabilityResponse {
+            in_active: 1.0.into(),
```

Comment:
I'd probably do 2 separate if checks here for each of the sets, because it's possible we'll have 100% to be in reserve, but not in the active set. For example, assume the following:
- we have 200 nodes
- reserve size is set to 300
- active set size is set to 100
- we have 100% to be in reserve set, but the same is not true for the active set